inst/include/Rcpp/sugar/functions/is_na.h

```diff
@@ -33,11 +33,11 @@ class IsNa : public ::Rcpp::VectorBase< LGLSXP, false, IsNa<RTYPE,NA,VEC_TYPE> >
 
 	IsNa( const BASE& obj_) : obj(obj_){}
 
-	inline int operator[]( int i ) const {
+        inline int operator[]( R_xlen_t i ) const {
```

Comment:
Sorry for the mistake. The original indention is tab and I used spaces here. Any preference for indention?